chromium_src/components/metrics/metrics_service_accessor.cc

```diff
@@ -1,13 +1,38 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#if defined(BRAVE_CHROMIUM_BUILD)
-#define GOOGLE_CHROME_BUILD
-#endif
+#include "components/metrics/metrics_service_accessor.h"
```

Comment:
this doesn't seem right, we only added GOOGLE_CHROME_BUILD for part of the file, I don't think we want it defined for the whole file